readthedocs/support/views.py

```diff
@@ -0,0 +1,144 @@
+"""Support views."""
+
+import base64
+import hashlib
+import hmac
+import logging
+
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.db.models import Q
+from rest_framework.response import Response
+from rest_framework.status import (
+    HTTP_400_BAD_REQUEST,
+    HTTP_500_INTERNAL_SERVER_ERROR,
+)
+from rest_framework.views import APIView
+
+from readthedocs.core.utils.extend import SettingsOverrideObject
+from readthedocs.support.front import FrontClient
+
+log = logging.getLogger(__name__)
+
+
+class FrontWebhookBase(APIView):
+
+    """
+    Front's webhook handler.
+
+    Currently we only listen to inbound messages events.
+    Contact information is updated when a new message is received.
+
+    See https://dev.frontapp.com/docs/webhooks-1.
```

Comment:
```suggestion
    See https://dev.frontapp.com/docs/webhooks-1
```

Make the link clickable 😄

Reply:
Works for me with the period (gnome-terminal and kitty)

Reply:
He, it doesn't work on GitHub 🙃